### ISSUE TITLE
Add an option to ignore emotes the same way you could in c1

### ISSUE THREAD

**Issue by GaZaTu** [Chatterino/chatterino2#867]
The current solution requires me to type the emote twice AND add some invisible character as a replacement.
I really don't think this is user friendly.
I would like to be able to use it the same way i was able to in c1.
Just a simple line separated list.
Some kind of import button that opens a window to do that would be perfectly fine aswell.

**Comment by pajlada** [comment]
Sounds good, I agree it's a bit convoluted to get the same functionality as in chatterino1

**Comment by hemirt** [comment]
twitch emote could be ignored [here](https://github.com/fourtf/chatterino2/blob/master/src/providers/twitch/TwitchMessageBuilder.cpp#L180) and bttv emote [here](https://github.com/fourtf/chatterino2/blob/master/src/providers/twitch/TwitchMessageBuilder.cpp#L987)

**Comment by Pashuu** [comment]
Adding an option when right clicking to ignore a single emote or all emotes from the same account would also do wonders

**Comment by alazymeme** [comment]
Isn't this possible using the ignores tab? Or am I missing something?

**Comment by apa420** [comment]
You can ignore emotes by using the ignores tab and setting the replacement to nothing.

**Comment by apa420** [comment]
Misunderstood the issue

**Comment by alazymeme** [comment]
> Misunderstood the issue.
> It has not been resolved.

What did you misunderstand? This works just fine by adding the emote in the ignores tab....

**Comment by apa420** [comment]
Removing the emote image but keeping the text doesn't work intuitively. Like op said, if you want to do it you'd have to add a zero-width char after.

**Comment by alazymeme** [comment]
> if you want to do it you'd have to add a zero-width char after.

I tested it by not adding the zero-width character and it worked just fine (even spamming it multiple times)

**Comment by apa420** [comment]
Yes, that's the point, it's a very dumb workaround for a feature that was in ch1.

**Comment by alazymeme** [comment]
How is it a dumb workaround? It works just fine by adding a single ignore line with the emote...
Literally this is all you need to do:
![image](https://user-images.githubusercontent.com/12804673/83325761-3b3db680-a2b2-11ea-9fb1-7892fcc53107.png)

**Comment by apa420** [comment]
In your case "gachiBASS" should appear as gachiBASS in chat (no emote). Not ***

**Comment by alazymeme** [comment]
No, it wouldn't as I explained in my previous comment: https://github.com/Chatterino/chatterino2/issues/867#issuecomment-636308607
This blocks ALL instances of it (including that WITH the zero-width character).

**Comment by apa420** [comment]
You're so dense. Read the thread again.

**Comment by alazymeme** [comment]
> You're so dense. Read the thread again. The point is to keep the text of the emote, but not the emote image.

I don't appreciate your childish comments. Based on the title, I was under the assumption that you gazatu wanted to ignore emotes.

**Comment by apa420** [comment]
I had to explain the same thing 4 times. I'm going to call you dense for it.

**Comment by alazymeme** [comment]
Yes and you failed to explain it well 3 times (likely due to a native language barrier). Keep the childish comments out of it.

**Comment by GaZaTu** [comment]
i feel like i have to explain this once and for all, the main issue is that its not userfriendly to well "disable" emote images, in c2 when you ignored an emote it replaced the image with its code. thats the functionality i was missing.
